src/plugins/custom_integrations/common/index.ts

```diff
@@ -12,43 +12,45 @@ export const PLUGIN_NAME = 'customIntegrations';
  * A map of category names and their corresponding titles.
  */
 // TODO: consider i18n
-export const INTEGRATION_CATEGORY_DISPLAY = {
-  aws: 'AWS',
-  azure: 'Azure',
-  cloud: 'Cloud',
-  config_management: 'Config management',
-  containers: 'Containers',
-  crm: 'CRM',
-  custom: 'Custom',
-  datastore: 'Datastore',
-  elastic_stack: 'Elastic Stack',
-  google_cloud: 'Google Cloud',
-  infrastructure: 'Infrastructure',
-  kubernetes: 'Kubernetes',
-  languages: 'Languages',
-  message_queue: 'Message queue',
-  microsoft_365: 'Microsoft 365',
-  monitoring: 'Monitoring',
-  network: 'Network',
-  notification: 'Notification',
-  os_system: 'OS & System',
-  productivity: 'Productivity',
-  security: 'Security',
-  sample_data: 'Sample data',
-  support: 'Support',
-  threat_intel: 'Threat intelligence',
-  ticketing: 'Ticketing',
-  version_control: 'Version control',
-  web: 'Web',
+export const INTEGRATION_CATEGORY_DISPLAY: {
+  [key: string]: { title: string; parent_id?: string };
+} = {
+  aws: { title: 'AWS', parent_id: undefined },
```

Comment:
Properties needed to be expanded to better handle the merge logic described in PR

The list here will be updated in https://github.com/elastic/kibana/pull/153216 for 8.8